src/AstTranslator.cpp

```diff
@@ -225,55 +225,80 @@ std::unique_ptr<RamRelation> AstTranslator::translateRelation(
 }
 
 std::unique_ptr<RamValue> AstTranslator::translateValue(const AstArgument* arg, const ValueIndex& index) {
-    std::unique_ptr<RamValue> val;
-    if (!arg) {
-        return val;
-    }
+    class ValueTranslator : public AstVisitor<std::unique_ptr<RamValue>> {
```

Comment:
The check for nullptr is gone now.
```suggestion
    if (arg == nullptr) {
        return std::unique_ptr<RamValue>();
    }
    class ValueTranslator : public AstVisitor<std::unique_ptr<RamValue>> {
```

Reply:
Do we know when a nullptr occurs? That feels not right.

Reply:
The failed tests were all related to aggregation, I think
semantic/aggregates2
semantic/bool
evaluation/aggregates
evaluation/average
evaluation/count_sccs1
evaluation/magic_aggregates
evaluation/magic_dominance

When AstTranslator reaches line 546  the AstAggregator has a null target expression.
```
// translate target expression
std::unique_ptr<RamValue> value = translateValue(cur->getTargetExpression(), valueIndex);
```
So, anything with a null target expression.

Reply:
I think that's anything that uses count